static/templates/stream_settings/stream_settings_overlay.hbs

```diff
@@ -26,6 +26,22 @@
                         <i class="fa fa-remove" aria-hidden="true"></i>
                     </button>
                 </div>
+                <div class="nothing-to-show">
+                    <div class="view_all_stream_button_container">
+                        <span>
+                            You are not subscribed to any streams. <a href="#streams/all">View all streams</a>
+                        </span>
+                    </div>
+                    <div class="create_stream_button_container">
+                        {{#if can_create_streams}}
+                        <span>
+                            {{#tr}}
+                                There are no streams you can view in this organization. <a class="create_new_stream_link" href="#streams/new">Create a stream</a>
+                            {{/tr}}
```

Comment:
This is not correctly tagged for translation. I can see error in console while opening the stream settings overlay. See https://zulip.readthedocs.io/en/latest/translating/internationalization.html#handlebars-templates.